backend/wmg/api/wmg-api.yml

```diff
@@ -89,6 +89,8 @@ paths:
                 is_rollup:
                   type: boolean
                   default: true
+                compare:
```

Comment:
@ashin-czi Can we add a comment here for the acceptable values please 🙏 ? E.g., `"sex", "self_reported_ethnicity", "disease"` Thank you!